Source/monster.h

```diff
@@ -127,6 +127,13 @@ struct CMonster {
 	/** placeflag enum as a flags*/
 	uint8_t mPlaceFlags;
 	AnimStruct Anims[6];
+	/**
+	 * @brief Returns AnimStruct for specified graphic

```

Comment:
Bit nitpicky but this should document the param and return despite those being pretty obvious from the name.

Reply:
I'm not a fan of inserting comments that doesn't help anyone.
If you really want it, I can add it.

Something like this?

```
	/**
	 * @brief Returns AnimStruct for specified graphic
	 * @param graphic Specifies which AnimStruct should be returned
	 * @return AnimStruct for the graphic
	 */
```

Reply:
I'm fine with leaving out lines that aren't helpful

Reply:
In that case no need to worry about this :)